### PR TITLE
lime 1.0.5 (new formula)

### PR DESCRIPTION
**Pull request by lschumm**
- [x] gone through contribution guide
- [x] `brew install --build-from-source lime` passes
- [x] `brew audit --strict lime` shows no errors

### CODE REVIEW

File: Formula/lime.rb [hunk 1]
```diff
@@ -0,0 +1,24 @@
+#
+# [lime.rb]
+#
+# The Homebrew installer for the Lime programming language.
+# Copyright (C) 2019, Liam Schumm & Andy Merrill
```
Comment (SMillerDev) [inline Formula/lime.rb]: Could you remove all the comments

File: Formula/lime.rb [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+#
+
+class Lime < Formula
+  include Language::Python::Virtualenv
+
+  desc "Experimental, stack-based, purely functional programming language"
+  homepage "https://limelang.xyz"
+  url "https://gitlab.com/lschumm/lime/-/archive/1.0.4/lime-1.0.4.tar.gz"
+  sha256 "307fc9742e2fd94bc124e73e7bd659a0e219309bd545a25e6b8aa119a4661bd1"
+  head "https://gitlab.com/lschumm/lime"
```
Comment (SMillerDev) [inline Formula/lime.rb]: New formulae shouldn't have a HEAD spec. Those are only for testing unreleased versions when it's really needed.

File: Formula/lime.rb [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+
+  def install
+    virtualenv_install_with_resources
+  end
+
+  test do
+    system "./test"
```
Comment (SMillerDev) [inline Formula/lime.rb]: Could you do something here to test the install?

**Review by SMillerDev — CHANGES_REQUESTED** [review]

**Comment by fxcoudert** [comment]
2 stars on Gitlab, not packaged in Linux distros as far as I can see, not on PyPI(?).

File: Formula/lime.rb (hunk 1 above)
Comment (lschumm) [reply Formula/lime.rb]: done in 0d934a8

File: Formula/lime.rb (hunk 2 above)
Comment (lschumm) [reply Formula/lime.rb]: done in 0d934a8

File: Formula/lime.rb (hunk 3 above)
Comment (lschumm) [reply Formula/lime.rb]: done in 0d934a8

**Comment by lschumm** [comment]
@SMillerDev, thanks a lot for the feedback. i've made all of your requested changes. `brew install --build-from-source`, `brew audit --strict`, and `brew test` all pass now.

**Comment by SMillerDev** [comment]
Your sha hash is wrong, this can't possibly have build from source correctly.

**Comment by lschumm** [comment]
@SMillerDev whoops, thanks for the catch. i updated the hash in my personal tap but forgot to do it in my `homebrew-core` fork.

**Comment by lschumm** [comment]
alrighty, closing until @BrewTestBot's notability guidelines are satisfied.

**Pull request closed without merging**